test.html

```diff
@@ -88,118 +115,132 @@
+        }
+        .second,
+        .third .data-block-header {
+          text-align: center;
+        }
+
+        .third .data-block {
+          flex: 1;
+          display: flex;
+          flex-direction: column;
+        }
+
+        .third .data-block .data-block-content {
+          flex: 1;
+        }
+
+        .second {
+          display: flex;
+        }
+
+        .left-side,
+        .right-side {
+          flex: 1
+        }
+
+        .second .right-side .data-block-content {
+          height: 145px;
         }
 
     </style>
 </head>
 <body>
     <div class="wrapper">
-    <div class="first">
-        <table>
-            <tr>
-                <td>
-                    <div class="data-block-header" id="part_r0">1071 </div><div class="data-block">VZ80A</div>
-                </td>
-            </tr>
-            <tr>
-                <td>
-                    <div class="data-block-header" id="part_r0">1071</div><div class="data-block">VZ80A</div>
-                </td>
-            </tr>
-            <tr>
-                <td>
-                    <div class="data-block-header" id="part_r0">1071</div><div class="data-block">AVZ80</div>
-                </td>
-            </tr>
-        </table>
-    </div>
-
-        <div class="second">
-            <table>
-                <tr>
-                    <td><div class="data-block-header" id="part_r0">1071s </div><div class="data-block">VZ80A</div></td>
-                    <td rowspan="2"><div class="data-block-header" id="part_r0">1071 </div><div class="data-block" style="height:62%">VZ80A</div></td>
-                </tr>
-                <tr>
-                    <td><div class="data-block-header" id="part_r0">1071 </div><div class="data-block">VZ80A</div></td>
-                </tr>
-            </table>
+      <div class="first">
+        <div class="data-block" id="part_r0">
+          <div class="data-block-header">1071</div>
+          <div class="data-block-content">VZ80A</div>
         </div>
-
-        <div class="third">
-            <table>
-                <tr>
-                    <td><div class="data-block-header" id="part_r0">1071</div><div class="data-block"><p class="vericaltext">VZ80Add</p></div></td>
-                    <td><div class="data-block-header" id="part_r0">1071</div><div class="data-block"><p class="vericaltext">VZ80Add</p></div></td>
-                    <td><div class="data-block-header" id="part_r0">1071</div><div class="data-block"><p class="vericaltext">VZ80Add</p></div></td>
-                </tr>
-            </table>
-
+        <div class="data-block" id="part_r0">
+          <div class="data-block-header">1071</div>
+          <div class="data-block-content">VZ80A</div>
+        </div>
+        <div class="data-block" id="part_r0">
+          <div class="data-block-header">1071</div>
+          <div class="data-block-content">VZ80A</div>
+        </div>
+      </div>
+      <div class="second">
+        <div class="left-side">
```

Comment:
you see, I had to seperate the `.second` div into `.left-side` and `.right-side`
Ideally, you wouldn't have to do this, but the layout we wanted to achieve is hard to achieve with just HTML..
Maybe it's possible using tables...
I am not really good at CSS and just tend to abuse flexbox because it's so convenient..
This is the best I could come up with